logic/interfuse/process_control_modifier.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+"""
+This file contains the Qudi interfuse between a process control and a process control.
```

Comment:
Guess this is some copy/paste error?

Reply:
Well no I don't think so. This interfuse connect to a process control and act like a process control.
Or am I missing something ?

Reply:
Ah well, yes you are right... it just looked weird.